app/models/sponsor.rb

```diff
@@ -11,14 +12,26 @@ class Sponsor < ActiveRecord::Base
 
   has_one :address
   has_many :chapters, through: :workshops
-  has_many :workshop_sponsors
-  has_many :workshops, through: :workshop_sponsors
+  has_many :contacts
```

Comment:
@despo this is where I lost the plot a little bit. Can you elaborate on how `Sponsors`, `Contacts`, `Members` and `MemberContacts` work together in this new system? Is `MemberContact` going to be removed eventually?

Reply:
There was a `Contact` model that had been created when the Sponsor-Contact functionality was originaly implemented but was never utilised. Instead, a member_contact association had been added and used, so up to now, a sponsor had many members as contacts, and could also have the details of a single contact stored directly on the Sponsor model using the `contact_first_name`, `contact_surname` and `email` fields.

I decided to completely drop the Sponsor/Model association (SponsorContacts) and instead use the existing Contact model to enable adding multiple independent contacts, as there is no benefit or need to have members associated with a Sponsor in the app.

After this PR is in I will be completely removing the MemberContact model, but at the moment I still need the association in place in order to correctly migrate existing data using the `sponsors:members:migrate_to_contacts` rake task.

So in short:
In this new functionality a `Sponsor` can have multiple `Contacts`. Nothing else!

Reply:
@despo ok cool, it's what I thought, thanks for clarifying :)